pkg/api/controller.go

```diff
@@ -2932,7 +2932,7 @@ func (c *Controller) GetSetupState(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 	state := setupStateNotInitialized
-	if initialized {
+	if initialized || c.Config.IsAuthTypeAPI() {
```

Comment:
Because initialized is set on setup, which is not used or needed when using  external authentication
The UI redirects to setup page in case initialized is false.
@nopcoder would you suggest a different way to skip this?